client/test/lib/onboarding.js

```diff
@@ -741,15 +742,15 @@ describe('Onboarding.Step', () => {
         it('should call overriding method', () => {
             // arrange
             let spy = sinon.spy();
-            let step = new Step({
-                isActive: spy
-            });
+            let defaultSpy = sinon.spy(Step.prototype, "isActive");
+            let step = new Step({ isActive: spy });
 
             // act
             let result = step.isActive();
 
             // assert
             assert(spy.calledOnce);
+            assert.equal(0, defaultSpy.callCount);
```

Comment:
This is already tested in test l.753 and should not be here. If you want, you may add a dedicated test for `prototype.isActive`, but test l.753 does pretty much the same.